docker-compose.yml

```diff
@@ -98,84 +97,109 @@ services:
+  # Manager for Kafka tasks    
+  stream-manager:
+    image: vstk/hercules-stream-manager:0.38.0-SNAPSHOT
+    container_name: stream-manager
     depends_on:
-      - kafka
-      - graphite
-      - elasticsearch
-    environment:
-      AIRLOCK_KAFKA_AUTO_OFFSET_RESET: Earliest
-    entrypoint: ["/bin/wait-for-it.sh", "elasticsearch:9200", "--timeout=30", "--strict", "--", "dotnet", "/app/Vostok.Airlock.Consumer.Logs/out/Vostok.Airlock.Consumer.Logs.dll"]
+      - init
+    ports:
+      - 6507:6507
+    volumes:
+      - ./etc/properties/stream-manager:/etc/hercules
 
-  # Metrics consumer
-  metrics:
-    image: vstk/airlock.consumer:latest
+  # API for creating Kafka tasks    
+  management-api:
+    image: vstk/hercules-management-api:0.38.0-SNAPSHOT
+    container_name: management-api
     depends_on:
-      - kafka
-      - graphite
-    environment:
-      AIRLOCK_KAFKA_AUTO_OFFSET_RESET: Earliest
-    entrypoint: ["/bin/wait-for-it.sh", "graphite:80", "--timeout=30", "--strict", "--", "dotnet", "/app/Vostok.Airlock.Consumer.Metrics/out/Vostok.Airlock.Consumer.Metrics.dll"]
+      - stream-manager
+    ports:
+      - 6307:6307
+    volumes:
+      - ./etc/properties/management-api:/etc/hercules
+  
+  # Init Kafka streams for logs, metrics and traces    
+  curl:
+    image: curlimages/curl:latest
+    command:
+      - sh
+      - /scripts/management.sh
+    volumes:
+      - ./etc/scripts:/scripts
+    depends_on:
+      - management-api
 
-  # Metrics aggregation consumer
-  metrics_aggr:
-    image: vstk/airlock.consumer:latest
+  # Kafka consumer for logs    
+  elastic-sink:
+    image: vstk/hercules-elastic-sink:0.38.0-SNAPSHOT
+    container_name: elastic-sink
     depends_on:
-      - kafka
-      - gate
-      - graphite
-    environment:
-      AIRLOCK_KAFKA_AUTO_OFFSET_RESET: Earliest
-    entrypoint: ["/bin/wait-for-it.sh", "graphite:80", "--timeout=30", "--strict", "--", "dotnet", "/app/Vostok.Airlock.Consumer.MetricsAggregator/out/Vostok.Airlock.Consumer.MetricsAggregator.dll"]
+      - elasticsearch
+      - management-api
+    volumes:
+      - ./etc/properties/elastic-sink:/etc/hercules
 
-  # Tracing consumer
-  tracing:
-    image: vstk/airlock.consumer:latest
+  # Kafka consumer for metrics    
+  graphite-sink:
+    image: vstk/hercules-graphite-sink:0.38.0-SNAPSHOT
+    container_name: graphite-sink
     depends_on:
-      - kafka
       - graphite
-      - cassandra
-    environment:
-      AIRLOCK_KAFKA_AUTO_OFFSET_RESET: Earliest
-    entrypoint: ["/bin/wait-for-it.sh", "cassandra:9042", "--timeout=30", "--strict", "--", "dotnet", "/app/Vostok.Airlock.Consumer.Tracing/out/Vostok.Airlock.Consumer.Tracing.dll"]
+      - management-api
+    volumes:
+      - ./etc/properties/graphite-sink:/etc/hercules
 
-  contrails_api:
-    image: vstk/contrails.api:latest
-    container_name: contrails_api
+  # Kafka consumer for traces    
+  tracing-sink:
+    image: vstk/hercules-tracing-sink:0.38.0-SNAPSHOT
```

Comment:
Hercules components version is duplicated 8 times. Is there a way to make it a single variable defined once?